knowledge-base/how-to-create-graph-with-merged-last-column-which-can-expand-and-collapse.md

```diff
@@ -52,7 +52,8 @@ In this [expression](../report-expressions), the value which will be set when th
 
 In this way, **+** will be shown when the columns are merged and **-** when they are expanded.
 
-4. Add a [Drillthrough Report Action](../designing-reports-interactivity-drill-through-report-links) to the textbox:  **Action** -> **Navigate to Report** -> **Select a report source** -> **URL or file** -> fill the path to the report ->**Edit Parameters:**
+4. Add a [Drillthrough Report Action](../designing-reports-interactivity-drill-through-report-links) to the textbox which will point to the same report (i.e. to itself):  **Action** -> **Navigate to Report** -> **Select a report source** -> **URL or file** -> fill the path to the report ->**Edit Parameters:**
  - **Parameter Name**: paramMerged
  - **Parameter Value**: = Parameters.paramMerged.Value = 'False'
 
+![](https://media.giphy.com/media/dVdEi7hpjbJWy0XR9Z/giphy.gif)
```

Comment:
This should not be hosted externally. Create a new folder inside images called resources and add it there.